### PR TITLE
Update rails to 6.0.3.2

### PR DESCRIPTION
**Pull request by pudinha**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -34,21 +36,15 @@
...
   # config.action_view.raise_on_missing_translations = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.hosts = [
+    "draft-router-api.dev.gov.uk",
+    "router-api.dev.gov.uk",
+  ]
```
Comment (pixeltrix) [inline config/environments/development.rb:49]: I think on reflection we should use `<<` so we preserve `127.0.0.1`, `::1` and `localhost` as valid hosts, e.g

``` ruby
config.hosts << "draft-router-api.dev.gov.uk"
config.hosts << "router-api.dev.gov.uk"
```

Comment (pudinha) [reply config/environments/development.rb:49]: true

Comment (pudinha) [reply config/environments/development.rb:49]: done

File: Gemfile [hunk 2]
```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 gem "gds-sso"
 gem "govuk_app_config"
 gem "plek"
-gem "rails", "~> 5.2"
+gem "rails", "~> 6.0.3"
```
Comment (pixeltrix) [inline Gemfile:6]: The default Rails Gemfile has this line:

``` ruby
gem "rails", "~> 6.0.3", ">= 6.0.3.2"
```

to ensure that security release is applied so we should follow that

Comment (pudinha) [reply Gemfile:6]: :sparkles:

**Pull request merged**